w3-hw1-vy/cypress/e2e/bookstore.cy.js

```diff
@@ -0,0 +1,57 @@
+import { bookstore } from "../pages/bookstore/bookstorePage";
+import { loginPage } from "../pages/bookstore/loginPage";
+import { profilePage } from "../pages/bookstore/profilePage";
+
+
+describe("Add book", () => {
+    beforeEach(() => {
+        cy.fixture("book").as("book");
+        cy.visit(Cypress.env("login"));
+    });
+
+    it("TC-ADD_BOOK-1: Add a book name Git Pocket Guide into my collection", () => {
+        cy.get("@book").then((book) => {
+            loginPage
```

Comment:
login and delete all books are precondition in add book cases, put them in `beforeEach()`